### ISSUE TITLE
Decode Column Name Byte Strings

### ISSUE THREAD

**Issue by bkief** [IBM/nzpy#32]
Decode Columns names from byte strings to python string: `col["name"].decode('utf8')`

https://github.com/IBM/nzpy/blob/b54702780c909e129c92533bf3be6ac62ab80cf1/nzpy/core.py#L807

**Comment by aniket-s-kulkarni** [comment]
@bkief yes, probably `col["name"].decode(self._client_encoding)` would better. Please raise a PR.
